edx_proctoring/templates/proctoring/seq_proctored_exam_error.html

```diff
@@ -0,0 +1,56 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Your proctoring session is in error
```

Comment:
How about something like: "There are problems with your proctoring session. Contact edX Support."?
@griffresch It would be helpful to provide an action and link here, do you agree? And is edX Support the right place? 
@chrisndodge If Griff agrees, we can link "edX Support" to https://www.edx.org/contact-us